test/controllers/DeleteAdditionalMetadataControllerSpec.scala

```diff
@@ -492,9 +568,15 @@ class DeleteAdditionalMetadataControllerSpec extends FrontEndTestHelper {
     val cancelButtonHref =
       s"/consignment/$consignmentId/additional-metadata/selected-summary/$metadataType?fileIds=${fileIds.mkString("&amp;")}"
     pageString must include(s"""form action="$deleteButtonHref"""")
+    if (!hasEnteredMetadata) {
+      pageString must include("deleteAdditionalMetadata.noMetadataWarningMessage")
+    } else {
+      pageString must (include("deleteAdditionalMetadata.closureDeletionWarningMessage") or include("deleteAdditionalMetadata.descriptiveDeletionWarningMessage"))
```

Comment:
This looks fine but it's better to add `metadataType` in the warming message:

        `pageString must include(s"deleteAdditionalMetadata.${metadataType}DeletionWarningMessage")`